tools/harness-thci/OpenThread.py

```diff
@@ -2038,7 +2038,7 @@ def getCommissioningLogs(self):
                         else PlatformDiagnosticPacket_Type.UNKNOWN
                 elif "len" in infoType:
                     EncryptedPacket.TLVsLength = int(infoValue)
-                    payloadLineCount = int(infoValue)/16 + 1
+                    payloadLineCount = int(infoValue)/16 if int(infoValue)%16 == 0 else int(infoValue)/16 + 1
```

Comment:
could `payloadLineCount = (int(infoValue) + 15)/16` replace the `if..else..`?
use a constant instead of the magic number 16?